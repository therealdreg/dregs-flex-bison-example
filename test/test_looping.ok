program test_write_natural
    natural n
    natural i
    natural f
    natural t
begin
    f := 0
    t := 3
    for i := f to t do
        write(69)
        write(i)
    done
    read(n)
    while not (n = 0) do
        i := n
        while i > 0 do
            write(n)
            i := i-1
        done
        read(n)
    done
end
