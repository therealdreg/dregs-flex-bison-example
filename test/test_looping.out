69
0
69
1
69
2
1
5
5
5
5
5
